gallery/src/PageHeader/PageHeader.scss

```diff
@@ -37,7 +37,7 @@
     min-width: max-content;
   }
 
-  &__version {
+  .nx-btn--tertiary {
```

Comment:
In this case I think it would be better to select the button either by an id or a very specific classname: we wouldn't want this style to apply to any other hypothetical tertiary button in the page header.

Reply:
> In this case I think it would be better to select the button either by an id or a very specific classname: we wouldn't want this style to apply to any other hypothetical tertiary button in the page header.

I added a classname here: [357cbcd](https://github.com/sonatype/sonatype-react-shared-components/pull/782/commits/357cbcda72bcf2e87d6c013223fa333f5a32ee8e)